index.html

```diff
@@ -38,23 +38,23 @@
     <h2 class="text-accent font-bold font-accent text-center text-3xl md:text-4xl lg:text-4xl">
       Made with Flame
     </h2>
-    <p class="max-w-sm text-center">
+    <p class="max-w-md text-center">
```

Comment:
this small size was breaking the line in a way that just the word Flame was on the second line